### PR TITLE
thunderbird: update 91.3.0

### PR DESCRIPTION
**Pull request by cgrzemba**
- add and comment some patches
	- remove mediator

### CODE REVIEW

File: components/mail/thunderbird/Makefile [hunk 1]
```diff
@@ -27,16 +27,20 @@
...
 include ../../../make-rules/shared-macros.mk
 
+# unfortunatly I see -m32 in CFLAGS despite that BUILD_BITS = 64 is set
+BITS = 64
+
 COMPONENT_NAME =	thunderbird
-COMPONENT_VERSION =	78.13.0
+COMPONENT_VERSION =	91.3.0
+COMPONENT_REVISION = 1
 COMPONENT_SUMMARY = Mozilla Thunderbird Email Application
```
Comment (AndWac) [inline components/mail/thunderbird/Makefile:38]: COMPONENT_REVISION=1 is wrong

File: components/mail/thunderbird/Makefile [hunk 2]
```diff
@@ -83,9 +90,12 @@ MOZCONFIG_DEBUG =	/dev/null
 MOZCONFIG_NON_DEBUG =	$(MOZCONFIG)
 endif
 
+COMPILER = gcc
+
```
Comment (AndWac) [inline components/mail/thunderbird/Makefile]: COMPILER = gcc is the default in shared-macros.mk and should not be necessary here.

File: components/mail/thunderbird/thunderbird.p5m [hunk 3]
```diff
@@ -32,16 +32,9 @@ license thunderbird.license license="MPL v2.0"
 file files/thunderbird.desktop path=usr/share/applications/thunderbird.desktop
 file usr/lib/$(MACH64)/thunderbird/chrome/icons/default/default48.png path=usr/share/pixmaps/thunderbird-icon.png
 
-link path=usr/bin/thunderbird target=../lib/amd64/thunderbird/thunderbird mediator=thunderbird mediator-version=78
-
```
Comment (AndWac) [inline components/mail/thunderbird/thunderbird.p5m:36]: You have removed too much - the link action needs to be kept, it's only the mediator attribute that has to be removed. We still need the link.

Comment (cgrzemba) [reply components/mail/thunderbird/thunderbird.p5m:36]: have the link readded.

Comment (3eka) [reply components/mail/thunderbird/thunderbird.p5m:36]: Version should be 91, right?

Comment (cgrzemba) [reply components/mail/thunderbird/thunderbird.p5m:36]: we remove the mediator, because installation of different versions is not intended

**Pull request merged**